src/app/modules/home/components/alert/alert.component.scss

```diff
@@ -0,0 +1,11 @@
+.alert {
+    text-align: center;
+    margin: 1rem auto;
+    width: 80%;
+  &__danger {
```

Comment:
In code bellow you use 'alert-danger', but here i can see only styles for 'alert__danger'